src/nodes/typeGuards/simple.ts

```diff
@@ -0,0 +1,247 @@
+export function isFunctionDeclaration(node: ts.Node): node is ts.FunctionDeclaration { return node.kind === ts.SyntaxKind.FunctionDeclaration; }
+export function isFunctionExpression(node: ts.Node): node is ts.FunctionExpression { return node.kind === ts.SyntaxKind.FunctionExpression; }
+export function isFunctionTypeNode(node: ts.Node): node is ts.FunctionTypeNode { return node.kind === ts.SyntaxKind.FunctionType; }
+export function isGetAccessorDeclaration(node: ts.Node): node is ts.GetAccessorDeclaration { return node.kind === ts.SyntaxKind.GetAccessor; }
+export function isHeritageClause(node: ts.Node): node is ts.HeritageClause { return node.kind === ts.SyntaxKind.HeritageClause; }
+export function isIdentifier(node: ts.Node): node is ts.Identifier { return node.kind === ts.SyntaxKind.Identifier; }
+export function isIfStatement(node: ts.Node): node is ts.IfStatement { return node.kind === ts.SyntaxKind.IfStatement; }
+export function isImportClause(node: ts.Node): node is ts.ImportClause { return node.kind === ts.SyntaxKind.ImportClause; }
+export function isImportDeclaration(node: ts.Node): node is ts.ImportDeclaration { return node.kind === ts.SyntaxKind.ImportDeclaration; }
+export function isImportEqualsDeclaration(node: ts.Node): node is ts.ImportEqualsDeclaration { return node.kind === ts.SyntaxKind.ImportEqualsDeclaration; }
+export function isImportExpression(node: ts.Node): node is ts.ImportExpression { return node.kind === ts.SyntaxKind.ImportKeyword; }
+export function isImportSpecifier(node: ts.Node): node is ts.ImportSpecifier { return node.kind === ts.SyntaxKind.ImportSpecifier; }
+export function isImportTypeAssertionContainer(node: ts.Node): node is ts.ImportTypeAssertionContainer { return node.kind === ts.SyntaxKind.ImportTypeAssertionContainer; }
+export function isImportTypeNode(node: ts.Node): node is ts.ImportTypeNode { return node.kind === ts.SyntaxKind.ImportType; }
+export function isIndexedAccessTypeNode(node: ts.Node): node is ts.IndexedAccessTypeNode { return node.kind === ts.SyntaxKind.IndexedAccessType; }
+export function isIndexSignatureDeclaration(node: ts.Node): node is ts.IndexSignatureDeclaration { return node.kind === ts.SyntaxKind.IndexSignature; }
+export function isInferTypeNode(node: ts.Node): node is ts.InferTypeNode { return node.kind === ts.SyntaxKind.InferType; }
+export function isInKeyword(node: ts.Node): node is ts.InKeyword { return node.kind === ts.SyntaxKind.InKeyword; }
+export function isInputFiles(node: ts.Node): node is ts.InputFiles { return node.kind === ts.SyntaxKind.InputFiles; }
+export function isInterfaceDeclaration(node: ts.Node): node is ts.InterfaceDeclaration { return node.kind === ts.SyntaxKind.InterfaceDeclaration; }
+export function isIntersectionTypeNode(node: ts.Node): node is ts.IntersectionTypeNode { return node.kind === ts.SyntaxKind.IntersectionType; }
+/** @deprecated Use isJSDoc */
+export function isJsDoc(node: ts.Node): node is ts.JSDoc { return isJSDoc(node); }
```

Comment:
I added this simply for compatibility with `tsutils`. But if we're not making this lib a "drop-in" replacement for `tsutils`, then yeah, this can be removed.

Reply:
I should modify or remove that line from the readme, heh